SPECS/cups/cups.spec

```diff
@@ -0,0 +1,3826 @@
+# increase the timeout
+Patch6:         cups-driverd-timeout.patch
+# usb backend didn't get any notification about out-of-paper because of kernel
+Patch7:         cups-usb-paperout.patch
+# uri compatibility with old Fedoras
+Patch8:         cups-uri-compat.patch
+# use IP_FREEBIND, because cupsd cannot bind to not yet existing IP address
+# by default
+Patch9:         cups-freebind.patch
+# add support of multifile
+Patch10:        cups-ipp-multifile.patch
+# prolongs web ui timeout
+Patch11:        cups-web-devices-timeout.patch
+# failover backend for implementing failover functionality
+# TODO: move it to the cups-filters upstream
+Patch12:        cups-failover-backend.patch
+# add device id for dymo printer
+Patch13:        cups-dymo-deviceid.patch
+#### UPSTREAM PATCHES (starts with 1000) ####
+##### Patches removed because IMHO they aren't no longer needed
+##### but still I'll leave them in git in case their removal
+##### breaks something.
+BuildRequires:  automake
+# gcc and gcc-c++ is no longer in buildroot by default
+# gcc for most of files
+BuildRequires:  gcc
+# gcc-c++ for ppdc and cups-driverd
+BuildRequires:  gcc-c++
+BuildRequires:  krb5-devel
+BuildRequires:  libacl-devel
+# make is used for compilation
+BuildRequires:  make
+BuildRequires:  openldap-devel
+BuildRequires:  pam-devel
+BuildRequires:  pkgconf-pkg-config
+# Make sure we get postscriptdriver tags.
+#BuildRequires: python3-cups
+BuildRequires:  systemd
+# needed for systemd rpm macros according FPG
+BuildRequires:  systemd-rpm-macros
+# needed for decompressing functions when reading from gzipped ppds
+BuildRequires:  zlib-devel
+BuildRequires:  pkgconfig(avahi-client)
+BuildRequires:  pkgconfig(dbus-1)
+BuildRequires:  pkgconfig(gnutls)
+BuildRequires:  pkgconfig(libsystemd)
+BuildRequires:  pkgconfig(libusb-1.0)
+Requires:       %{name}-client = %{version}-%{release}
+Requires:       %{name}-filesystem = %{version}-%{release}
+Requires:       %{name}-libs = %{version}-%{release}
+# We ship udev rules which use setfacl.
+Requires:       acl
+Requires:       dbus
+Requires:       systemd
+Requires(post): grep
+Requires(post): sed
+Requires(post): systemd
+Requires(postun): systemd
+# Requires working PrivateTmp (bug #807672)
+Requires(pre):  systemd
+Requires(preun): systemd
+# avahi is needed for mDNS discovery and sharing queues
+Recommends:     avahi
+# getaddrinfo from glibc needs nss-mdns or systemd-resolved for resolving
+# mdns .local addresses. Don't require a specific package for now and let
+# the user to decide what to use
+# just recommend nss-mdns for Fedora for now to have working default, but
+# don't hardwire it for resolved users
+Recommends:     nss-mdns
+%if %{lspp}
+BuildRequires:  audit-libs-devel
+BuildRequires:  libselinux-devel
+%endif
+# Make sure we have some filters for converting to raster format.
+%if 0%{with missing_dependencies}
+Requires:       cups-filters
+%endif
+
+%package        client
```

Comment:
Is there a way to tell the linter to keep the `%package` and `%description` sections together for each subpackage? It would be good to stick to what we're doing in most (all?) other specs.